docs/others/obj_property.rst

```diff
@@ -35,6 +37,8 @@ Two APIs are provided to get/set widget properties. It can be enabled by setting
     lv_result_t lv_obj_set_property(lv_obj_t * obj, const lv_property_t * value);
     lv_property_t lv_obj_get_property(lv_obj_t * obj, lv_prop_id_t id);
 
+    lv_prop_id_t lv_obj_property_name2id(const lv_obj_class_t * clz, const char * name);
```

Comment:
```suggestion
    lv_prop_id_t lv_obj_get_property_id(const lv_obj_class_t * clz, const char * name);
```

Reply:
I recall that micropython has a naming constraint that the first parameter for lv_obj_xxx APIs should be `lv_obj_t *`.
Is it still true?

`lv_obj_get_` sounds like to get something from an object. But here it intends to convert a property name to ID based on its class. How about `lv_obj_property_get_id`?

Reply:
I've just copied the name from the [code](https://github.com/lvgl/lvgl/pull/6329/files#diff-c9ad3a7d72566d008dd5214f92aae68ce51e57b0aa2de961cc373e448698f9aaR174). :slightly_smiling_face: 

But you are right, the `lv_obj_property` prefix seems more appropriate.